.github/workflows/report.yaml

```diff
@@ -0,0 +1,64 @@
+name: Generate Data Usage Report
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Should we also run this on a weekly basis?

Reply:
It's on PR push just so I can test easily, but yes, 1/week sounds good to me. @kabilar Do you have a preference for what day/time?

Reply:
Great, thank you.  How about Monday at 6am EST?  We can then review the report on Monday mornings.